web_url/__openerp__.py

```diff
@@ -7,7 +12,7 @@
         You can send feedback to info@emiprotechnologies.com,
```

Comment:
Looks like this module is from another company.
Can you double check: if that is the case then close this PR